### PR TITLE
new nextcloud_configuration.sh

### LINKED ISSUE #1063: Rich workspaces
I have tried the Nextcloud 18 release a bit and I really don't like the rich workspaces, because I think they are really confusing for most people and can spam your file structure with readme files.
So I would like to provde an option during the startup script to disable them, based on the command:
`occ_command config:app:set text workspace_available --value=0`
this will only work starting at 18.0.1: https://github.com/nextcloud/text/issues/586#issuecomment-578387536

### PR DESCRIPTION
**Pull request by szaimen**
successor to https://github.com/nextcloud/vm/pull/1070
fix #1063
and #1052

### CODE REVIEW

**Comment by szaimen** [comment]
I think this is ready.

File: nextcloud-startup-script.sh [hunk 1]
```diff
@@ -394,14 +394,13 @@ case *"$choice" in
...
-    *"CookieLifetime"*)
-        clear
-        run_static_script cookielifetime
-    ;;&
     *)
     ;;
 esac
 
+# Nextcloud configuration
+bash $SCRIPTS/configuration.sh
```
Comment (szaimen) [inline nextcloud-startup-script.sh]: Maybe we could move that inside the startup script if it is not that important?
Could maybe be after lets encrypt and directly before additional_apps?

Comment (szaimen) [reply nextcloud-startup-script.sh]: I moved it to where I think it fits best.

**Comment by enoch85** [comment]
OK, I'll merge this. Let's see if it produces any bugs.

**Pull request merged**